src/rolch/distributions/normal.py

```diff
@@ -1,25 +1,31 @@
+from typing import Tuple, Union
+
 import numpy as np
 import scipy.stats as st
 
-from rolch.base import Distribution
+from rolch.base import Distribution, LinkFunction
 from rolch.link import IdentityLink, LogLink
 
 
 class DistributionNormal(Distribution):
     """Corresponds to GAMLSS NO() and scipy.stats.norm()"""
 
-    def __init__(self, loc_link=IdentityLink(), scale_link=LogLink()):
-        self.n_params = 2
-        self.loc_link = loc_link
-        self.scale_link = scale_link
-        self.links = [self.loc_link, self.scale_link]
-
-    def theta_to_params(self, theta):
+    def __init__(
+        self,
+        loc_link: LinkFunction = IdentityLink(),
+        scale_link: LinkFunction = LogLink(),
+    ):
```

Comment:
We should add -> None also here, right?

```suggestion
    ) -> None:
```